### PR TITLE
Holocalls

### PR DESCRIPTION
**Pull request by Cyberboss**
https://www.youtube.com/watch?v=n-vrCAsdJs8&feature=youtu.be
:cl:
add: You can now make holocalls! Simply stand on a pad, bring up the menu, and select the holopad you wish to call. Remain still until someone answers. When they do, you'll be able to act just like an AI hologram until the call ends
/:cl:

TODO: Someone get me dialtone and ringing sounds

### CODE REVIEW

**Comment by CakeQ** [comment]
pls finish

**Comment by Cyberboss** [comment]
It's done, can't open till the freeze is over

File: code/datums/holocall.dm [hunk 1]
```diff
@@ -0,0 +1,154 @@
+#define HOLOPAD_MAX_DIAL_TIME 200
```
Comment (duncathan) [inline code/datums/holocall.dm]: Why not make this a var on `/datum/holocall` so that it can be VV'd?

Comment (duncathan) [inline code/datums/holocall.dm]: Might be best for it to be a `var/global` since I doubt there'd be many reasons to ever edit a single instance

**Comment by Cyberboss** [comment]
This could use a dialtone sound if anyone has ideas

File: code/datums/holocall.dm (hunk 1 above)
Comment (Cyberboss) [inline code/datums/holocall.dm]: It's a define because it's also used by the hologram machine.

**Comment by lzimann** [comment]
PRs that violated the freeze have to wait a week longer to be PRd or else people will keep opening their feature PRs during freezes

**Comment by Cyberboss** [comment]
I can already see a way around that for the next freeze, you memes

**Comment by lzimann** [comment]
the way around: don't open feature prs in the middle of the freeze

**Comment by Cyberboss** [comment]
>push branch
>don't open pr

ez

**Comment by lzimann** [comment]
But that's pretty much what I said.

**Comment by Cyberboss** [comment]
Just makes my life harder bc ya gotta remember the branch exists

**Comment by PKPenguin321** [comment]
Scree scree give pr

**Comment by Cyberboss** [comment]
@lzimann THE PEOPLE WANT CAKE

**Comment by kevinz000** [comment]
time to holocall my waifu

**Pull request merged**